### PR TITLE
Improve CGAL section

### PR DESCRIPTION
**Pull request by fdrmrc**
I addressed @peterrum 's comment ( https://github.com/dealii/release-papers/pull/78#pullrequestreview-1005653075). 

@simonsticko Do you think there's something I should add/highlight about the difference with `QuadratureGenerator` class?

### CODE REVIEW

File: 9.4/paper.tex [hunk 1]
```diff
@@ -775,19 +769,23 @@ \subsection{Integration with the Computational Geometry Algorithms
...
-    \begin{subfigure}[b]{0.5\textwidth}
+    \begin{subfigure}[b]{0.3\textwidth}
       \centering
-      \includegraphics[height=.2\paperheight]{png/cube_sphere_remeshed.png}
+      \includegraphics[height=.14\paperheight]{png/cube_sphere_remeshed.png}
       \caption{ \label{fig:corefinement_remeshed}}
     \end{subfigure}
-    \caption{\it (a) Union of the unit cube with an hyper ball. Notice
-      how the corefinement around the intersection results in badly shaped elements. (b) The remeshed \dealii{} Triangulation.}
+    \caption{\it (a) \texttt{Triangulation<3>} of a heart surface starting from an implicit function $f$. (b) Union of the unit cube with an hyper ball. Notice how the corefinement around the intersection gives badly shaped elements. (c) The remeshed \dealii{} Triangulation.}
```
Comment (drwells) [inline 9.4/paper.tex:788]: With three figures the rightmost one spills out into the margin a bit - we could probably fix that by trimming whitespace from the images.

**Comment by bangerth** [comment]
Let's not go around too many times via patch review. I'll just fix that myself in a second.

**Comment by peterrum** [comment]
@fdrmrc Thanks for the modifications. I have done some modification myself. Could you check these?

**Comment by fdrmrc** [comment]
Looks perfect to me, thanks @peterrum

**Pull request merged**